akka-docs/src/test/scala-jdk9-only/docs/stream/operators/source/AsSubscriber.scala

```diff
@@ -5,12 +5,12 @@
 package docs.stream.operators.source;
```

Comment:
another docs package that shouldn't be touched?

Reply:
Will check then.